subscription_oca/tests/test_subscription_oca.py

```diff
@@ -506,6 +506,21 @@ def test_subscription_oca_sub8_workflow(self):
         subscription.generate_invoice()
         subscription.template_id.invoicing_mode = "invoice_send"
         subscription.generate_invoice()
+        self.assertTrue(
+            len(
+                self.env["account.move"].search(
```

Comment:
@mondot You can do :

```suggestion
                self.assertEqual(
                    1,
                    len(subsription.invoice_ids.filtered("is_move_sent"))
                )
```

instead.

Reply:
Great, thanks for the feedback :)